tests/driver.sh

```diff
@@ -592,6 +592,39 @@ int main()
 }
 EOF
 
+# Multiplication for signed integers
+try_output 0 "35 -35 -35 35" << EOF
+int main()
+{
+    printf("%d %d %d %d\n", 5 * 7, 5 * -7, -5 * 7, -5 * -7);
```

Comment:
Always use `(-5)` and `(-7)`.

Reply:
Does it mean that I should enclose each negative integer in brackets?
```shell
# Multiplication for signed integers
try_output 0 "35 -35 -35 35" << EOF
int main()
{
    printf("%d %d %d %d\n", 5 * 7, 5 * (-7), (-5) * 7, (-5) * (-7));
    return 0;
}
EOF
```
Regardless, I will continue to work it tomorrow.

Reply:
> Does it mean that I should enclose each negative integer in brackets?

Yes, it is more friendly.